app/templates/main/index.html

```diff
@@ -1,8 +1,11 @@
 {% extends 'layouts/base.html' %}
-
+{# Some special functionality here: current_user represents the current user. is_authticated is false for an AnonymousUser
```

Comment:
Is this a comment that leaked form your documentation branch? The code below is self-documenting imo.